index.bs

```diff
@@ -267,11 +268,14 @@ available controls on the Picture-in-Picture window.
 
 ## Interaction with Page Visibility ## {#page-visibility}
 
+When {{pictureInPictureElement}} is set, the Picture-in-Picture window MUST
+be visible, even when <a>Document</a> is not in focus or hidden.
+
 The [[Page-Visibility]] specification defines a {{Document/visibilityState}}
-attribute used to determine the visibility state of a top level browsing
-context. For the purpose of Picture-in-Picture, the {{Document/visibilityState}}
-MUST always return "visible" when {{pictureInPictureElement}}
-is set and the Picture-in-Picture window is visible.
+attribute used to determine the visibility state of a <a>top-level browsing
+context</a>. For the purpose of Picture-in-Picture, the
+{{Document/visibilityState}} MUST always return "visible" when
+{{pictureInPictureElement}} is set.
```

Comment:
Looks good from the visibility state perspective. It's not clear in the spec which mechanisms users will typically have to eventually close such web sites, and how those mechanisms interact with the "exit pip" algorithm. Would be good to add some wording on that front (maybe a note would be enough).

Reply:
Yeah, just to say that browsers should make sure there is a way for users to eventually close that window (and it's not a way for annoying sites/ads to remain in the user's face forever)

Reply:
Done.